<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoffeeApp</class>
 <widget class="QMainWindow" name="CoffeeApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о кофе</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Название сорта</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Степень обжарки</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Молотый/в зернах</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Описание вкуса</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цена</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Объем упаковки</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
